Command/H5pBundleIncludeAssetsCommand.php

```diff
@@ -25,67 +22,85 @@ public function __construct(KernelInterface $appKernel)
     protected function configure()
     {
         $this
-            ->setDescription('Include the assets from the h5p vendor bundle in the public resources directory of this bundle.')
+            ->setDescription('Include the assets from the official h5p vendor bundle and our js script in the public resources directory.')
             ->addOption('copy', 'c', InputOption::VALUE_NONE, 'Copy files')
         ;
     }
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $this->includeAssets($input->getOption('copy') ?? false);
+        $copy = $input->getOption('copy') ?? false;
+        $this->includeH5PVendorAssets($copy);
+        $this->includeSfScript($copy);
 
         return 0;
     }
 
-    private function includeAssets(bool $copy)
+    private function getTargetDir(): string
     {
-        //get dir of vendor H5P
+        return $this->appKernel->getProjectDir().'/public/bundles/studith5p/h5p/';
+    }
 
+    private function includeH5PVendorAssets(bool $copy): void
+    {
         $fromDir = $this->appKernel->getProjectDir()."/vendor/h5p/";
-        //call service
-        //$toDir = $this->appKernel->getProjectDir().'/vendor/studit/h5p-bundle/public/h5p/';
-        $toDir = $this->appKernel->getProjectDir().'/public/bundles/studith5p/h5p/';
-
-        //$toDir = new FileLocator($fromDir);
-        // $test = $fileLocator->locate('@StuditH5PBundle');
-
-        //$toDir = $fileLocator->locate();
 
         $coreSubDir = "h5p-core/";
         $coreDirs = ["fonts", "images", "js", "styles"];
-        $this->createFiles($fromDir, $toDir, $coreSubDir, $coreDirs, $copy);
+        $this->createFiles($fromDir, $this->getTargetDir(), $coreSubDir, $coreDirs, $copy);
 
         $editorSubDir = "h5p-editor/";
         $editorDirs = ["ckeditor", "images", "language", "libs", "scripts", "styles"];
-        $this->createFiles($fromDir, $toDir, $editorSubDir, $editorDirs, $copy);
+        $this->createFiles($fromDir, $this->getTargetDir(), $editorSubDir, $editorDirs, $copy);
+    }
 
+    private function includeSfScript(bool $copy): void
+    {
+        $filename = "symfony-h5p.js";
+        $from = $this->appKernel->getProjectDir()."/vendor/jorisdugue/h5p-bundle/public/h5p/".$filename;
+        $to = $this->getTargetDir().$filename;
+
+        $copy
+            ? copy($from, $to)
+            : symlink($from, $to);
     }
 
-    private function createFiles($fromDir, $toDir, $subDir, $subDirs, $copy)
+    private function createFiles(string $fromDir, string $toDir, string $subDir, array $subDirs, bool $copy): void
     {
         foreach ($subDirs as $dir) {
             $src = $fromDir . $subDir . $dir;
             $dist = $toDir . $subDir . $dir;
+            mkdir($dist, 0777, true);
```

Comment:
Replace this for security

```suggestion
            mkdir($dist, 0750, true);
```